tesseract/tesseract_planning/tesseract_motion_planners/include/tesseract_motion_planners/hybrid/descartes_trajopt_array_planner.h

```diff
@@ -25,7 +25,7 @@ class DescartesTrajOptArrayPlanner : public MotionPlanner
    * @return True if successful otherwise false
    */
   bool setConfiguration(const DescartesMotionPlannerConfig<FloatType>& descartes_config,
-                        const TrajOptArrayPlannerConfig& trajopt_config);
+                        const TrajOptPlannerConfigBase::Ptr& trajopt_config);
```

Comment:
I recommend renaming `TrajOptPlannerConfigBase` to `TrajOptPlannerConfig` which is the convention throughout the existing code base. Then the implementation differentiates in the naming.